crd-generator/api/src/main/java/io/fabric8/crd/generator/v1/decorator/EnsureSingleStorageVersionDecorator.java

```diff
@@ -34,7 +34,7 @@ public EnsureSingleStorageVersionDecorator(String name) {
 
 	@Override
 	public void andThenVisit(CustomResourceDefinitionSpecFluent<?> spec, ObjectMeta resourceMeta) {
-    Predicate<CustomResourceDefinitionVersionBuilder> hasStorageVersion = CustomResourceDefinitionVersionFluentImpl::isStorage;
+    Predicate<CustomResourceDefinitionVersionBuilder> hasStorageVersion = CustomResourceDefinitionVersionFluentImpl::getStorage;
```

Comment:
I'm worried about this, have the boolean getters changed from isXXX to getXXX after the Sundr.io upgrade?

Reply:
I think that was a semantic change that @iocanel did, unrelated to the upgrade